DataManagementSystem/Agent/RequestOperations/RemoveFile.py

```diff
@@ -55,7 +55,7 @@ def __init__( self, operation = None, csPath = None ):
     gMonitor.registerActivity( "RemoveFileFail", "Failed file removals",
                                "RequestExecutingAgent", "Files/min", gMonitor.OP_SUM )
     # # re pattern for not existing files
-    self.reNotExisting = re.compile( "(no|not) such file or directory", re.IGNORECASE )
+    self.reNotExisting = re.compile( "(no|not) such file.*", re.IGNORECASE )
```

Comment:
Well, in this particular case, it doesn't really matter, but in general it's better to use raw strings (for example if you have \n in there... (see the beginning of the re module doc)